src/metabase/query_processor/streaming/json.clj

```diff
@@ -16,23 +16,32 @@
    :headers      {"Content-Disposition" (format "attachment; filename=\"query_result_%s.json\""
                                                 (u.date/format (t/zoned-date-time)))}})
 
+(defn- select-indices
```

Comment:
I redefined this helper function in each streaming namespace because I wasn't sure where it should go. Is it general enough to go somewhere like `metabase.util`?